lib/pseudolocalization.rb

```diff
@@ -5,9 +5,12 @@ module Pseudolocalization
   module I18n
     class Backend
       attr_reader :original_backend
+      attr_accessor :ignores
 
       def initialize(original_backend)
         @original_backend = original_backend
+        @ignores = []
+        yield self if block_given?
```

Comment:
This allows us to use a block to configure the ignores instead of accessing `I18n.backend` directly after initialization.

Reply:
I like the flexibility this gives. Can we add a test for this block style configuration as well?

Reply:
Added